libcst/codemod/_cli.py

```diff
@@ -756,6 +757,15 @@ def parallel_exec_transform_with_prettyprint(  # noqa: C901
 
         warnings += len(result.transform_result.warning_messages)
 
+        # Join the process to free any related resources.
+        # Remove all references to the process to allow the GC to
+        # clean up any file handles.
+        process = filename_to_process.pop(result.filename, None)
+        if process:
+            process.join()
+            if process in joinable_processes:
+                joinable_processes.remove(process)
```

Comment:
`joinable_processes.discard(process)` is a "branchless" way to accomplish the same thing